package.json

```
@@ -9,35 +10,41 @@
     "clone",
     "recursive"
   ],
-  "version": "4.2.2",
   "homepage": "https://github.com/TehShrike/deepmerge",
   "repository": {
     "type": "git",
     "url": "git://github.com/TehShrike/deepmerge.git"
   },
+  "license": "MIT",
   "main": "dist/cjs.js",
-  "engines": {
-    "node": ">=0.10.0"
-  },
+  "files": [
+    "/dist",
+    "index.d.ts",
+    "changelog.md",
+    "license.txt",
+    "readme.md"
+  ],
   "scripts": {
     "build": "rollup -c",
+    "size": "npm run build && terser --compress --mangle -- ./dist/umd.js | gzip -c | wc -c",
     "test": "npm run build && tape test/*.js && jsmd readme.md && npm run test:typescript",
-    "test:typescript": "tsc --noEmit test/typescript.ts && ts-node test/typescript.ts",
-    "size": "npm run build && uglifyjs --compress --mangle -- ./dist/umd.js | gzip -c | wc -c"
+    "test:typescript": "tsc --noEmit test/typescript.ts && ts-node test/typescript.ts"
   },
+  "dependencies": {},
   "devDependencies": {
+    "@rollup/plugin-commonjs": "^16.0.0",
+    "@rollup/plugin-node-resolve": "^10.0.0",
     "@types/node": "^8.10.54",
     "is-mergeable-object": "1.1.0",
-    "is-plain-obj": "^2.0.0",
+    "is-plain-obj": "^3.0.0",
     "jsmd": "^1.0.2",
-    "rollup": "^1.23.1",
-    "rollup-plugin-commonjs": "^10.1.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
-    "tape": "^4.11.0",
-    "ts-node": "7.0.1",
-    "typescript": "=2.2.2",
-    "uglify-js": "^3.6.1"
+    "rollup": "^2.33.1",
+    "tape": "^5.0.1",
+    "terser": "^5.5.1",
+    "ts-node": "^9.0.0",
+    "typescript": "^4.1.2"
   },
-  "license": "MIT",
-  "dependencies": {}
+  "engines": {
+    "node": ">=12"
```

node 10 is still in maintenance mode, was there some other reason to specify 12+?
```suggestion
    "node": ">=10"
```